### ISSUE TITLE
Cursor should jump to Leo chat box by default

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#40410]
### Description

While small this could be a very easily implemented quality of life improvement for Leo. Simply change the behavior so that when the Leo side panel is opened, cursor focus jumps to the chat box so that users can start typing immediately after the panel is opened.

I believe there is an argument here for including this behavior _only_ when the keyboard shortcut is used (`Ctrl + B` or `Cmd + B`) but imo any time the Leo panel is opened specifically we should move the focus there, instead of the user having to click into it.

### Steps to reproduce

N/A

### Actual result

N/A

### Expected result

N/A

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

v1.68.137

### Channel information

- [X] release (stable)
- [X] beta
- [X] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by zunjack** [comment]
I would honestly be using Leo more often if you implemented this very small quality of life change.

**Comment by Archie-2021** [comment]
I hope this implemented soon, it is a very annoying how it is right now and makes no sense. 

I think it would be nice idea to add a new hotkey to jump back to main window.
Right now there is only focus next/prev  keys , using them is not that fast (for some reason address bar also is in the list of these panes , it has its own hotkey to focus to). Maybe these issues should have their own FR.

**Comment by sudoshindo** [comment]
Seems to be already resolved?

**Comment by iV1n5** [comment]
+1
